<launch>
		<param name="/use_sim_time" value="true" />
		
		<node name="rosbag" pkg="rosbag" type="play" args=" -l --clock /home/ak/Git/new_rectified.bag" />
		
		<node name="image_proc" pkg="image_proc" type="image_proc" respawn="false" ns="/sensors/camera">
			<remap from="image_raw" to="image_rect_color" />
		</node>

		<node name="tf1" pkg="tf" type="static_transform_publisher" args="0.8 -0.35 0 1.5708 3.14159 1.5708 camera_optical velodyne 10" />

		<node name="tf2" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 velodyne rgb_output 10" />

		<node name="rgb_pointcloud" pkg="lidar_calib" type="rgb_pointcloud" args=""/>

</launch>